src/selectors/publish.js

```diff
@@ -16,6 +16,7 @@ import { serializeProfile } from '../profile-logic/process-profile';
 import {
   sanitizePII,
   getShouldSanitizeByDefault as getShouldSanitizeByDefaultImpl,
+  type SanitizeProfileResult,
```

Comment:
Oh, that's the first time I see mixed usage like that

Reply:
I was lazy.